media-gfx/synnefo/synnefo-9999.ebuild

```diff
@@ -4,45 +4,29 @@
 
 EAPI=6
 
+EGIT_REPO_URI="https://github.com/oyranos-cms/Synnefo.git"
 inherit cmake-utils
+[[ ${PV} == "9999" ]] && inherit git-r3
 
 DESCRIPTION="Qt front end for the Oyranos Color Management System"
 HOMEPAGE="https://github.com/oyranos-cms/Synnefo"
-if [[ ${PV} == "9999" ]] ; then
-	EGIT_REPO_URI="https://github.com/oyranos-cms/Synnefo.git"
-	inherit git-r3
-	KEYWORDS=""
-else
-	SRC_URI="https://github.com/oyranos-cms/Synnefo/archive/${PV}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
-fi
+[[ ${PV} == 9999 ]] || \
+SRC_URI="https://github.com/oyranos-cms/Synnefo/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD-2"
 SLOT="0"
-IUSE="qt5"
+[[ ${PV} == 9999 ]] || \
```

Comment:
Works for me. Please double check and if it works for you change it to a one liner.